docs/translations.md

```diff
@@ -0,0 +1,32 @@
+# Translation and Localization
+Translations are maintained in separate repositories that are managed by the
+[Mozilla Localization Team](https://github.com/mozilla-l10n). There is a
+Pontoon project for the
+[Relay Website](https://pontoon.mozilla.org/projects/firefox-relay-website/)
+and for the
+[Add-on](https://pontoon.mozilla.org/projects/firefox-relay-add-on/). More
+than 20 languages are supported.
+
+The translation repositories are included as submodules, such as
+`privaterelay/locales`. The submodule refers to a specific commit in the
+separate repository. A GitHub action periodically updates this to the latest
+commit, bringing in any new translations or other changes from the localization
+team.
+
+The translation bundles use the [Fluent format](https://projectfluent.org/).
+They are also included in the Docker image, and read by the email processing
```

Comment:
Maybe add something like

```suggestion
They are also included in the Docker image that is deployed to the stage and production environments, and read by the email processing
```

just so people know what that Docker image is used for? (I only recently learned this, and before that might be confused as to why this is relevant.)